packages/app/server/services/vite-service.ts

```
@@ -0,0 +1,70 @@
+	readonly #viteDevServer: ViteDevServer
+
+	private constructor(viteDevServer: ViteDevServer) {
+		this.#viteDevServer = viteDevServer
+	}
+
+	/**
+	 * Retrieves the instance of the ViteDevServer.
+	 *
+	 * @return {ViteDevServer} The ViteDevServer instance associated with the current object.
+	 */
+	public get viteDevServer(): ViteDevServer {
```

Biome (1.9.4)

[error] The public modifier is disallowed.

Remove the accessibility modifier.

(lint/nursery/useConsistentMemberAccessibility)